index.d.ts

```diff
@@ -26,23 +27,18 @@ interface HotKeysProps extends FocusTrapProps<HotKeys> {
   /**
    * A mapping of action names to handler functions
    */
-  handlers?: { [key: string]: (keyEvent: KeyboardEvent) => void };
+  handlers?: { [key: string]: (keyEvent?: KeyboardEvent) => void };
```

Comment:
Thanks, this was an oversight on my part. The `keyEvent` is indeed optionally accepted by the handler.

Reply:
That's cool, I don't even think I was aware of that, I'll be able to do some refactoring in my own project with that knowledge.

Is that documented? All the examples I saw used parameterless handler functions

Reply:
Maybe, I am mis-understanding you? All of the [examples for defining handlers](https://github.com/greena13/react-hotkeys#defining-handlers) accept the `KeyboardEvent` object.

Reply:
I'm looking in the `/master/index.js` file and not seeing them. But maybe I'm blind :-)